docs/docs/tutorials/securing-pages-and-api-routes.md

```diff
@@ -42,7 +42,7 @@ export default function Page() {
 
 ### Next.js (Middleware)
 
-With NextAuth.js 4.2.0 and Next.js 12, you can now protect your pages via the middleware pattern more easily. If you would like to protect all pages, you can create a `middleware.js` file in your root `pages` directory which looks like this:
+With NextAuth.js 4.2.0 and Next.js 12, you can now protect your pages via the middleware pattern more easily. If you would like to protect all pages, you can create a `middleware.js` file in at the root or in the `src` directory (same level as your `pages`) which looks like this:
```

Comment:
```suggestion
With NextAuth.js 4.2.0 and Next.js 12, you can now protect your pages via the middleware pattern more easily. If you would like to protect all pages, you can create a `middleware.js` file in the root `pages` directory or in the `src` directory at the same level as `pages`,  which looks like this:
```

Reply:
I just used the same language as the nextjs middleware docs here: https://nextjs.org/docs/advanced-features/middleware#using-middleware as I was originally putting the file in the `pages` directory and it was having no effect.